pkg/fspkg/bundle_package.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2022 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fspkg
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/cert-manager/trust-manager/pkg/util"
+)
+
+type PackageType string
+
+const (
+	// PackageTypeStatic indicates that the package does not change once loaded. This is the default.
+	// Currently, only static packages are supported.
+	PackageTypeStatic PackageType = "static"
+)
+
+// Package specifies the structure of JSON packages which can be read from a the filesystem
+// of the trust-manager container and subsequently used in a Bundle resource
+type Package struct {
+	// Name contains a friendly name for the bundle
+	Name string `json:"name"`
+
+	// Bundle contains the PEM-formatted certificates which are provided by this bundle.
+	Bundle string `json:"bundle"`
+
+	// Version identifies the bundle's version, to distinguish updated bundles from older counterparts
+	Version string `json:"version"`
```

Comment:
I don't think this is currently being exposed on the `Bundle` status when it has successfully synced. I think that is quite important.

Reply:
Yeah that's a useful review comment, thanks. I'll look into it!